diff --git a/sy-tomato-plugin/src/libs/user.ts b/sy-tomato-plugin/src/libs/user.ts
--- a/sy-tomato-plugin/src/libs/user.ts
+++ b/sy-tomato-plugin/src/libs/user.ts
@@ -27,7 +27,8 @@ export function resetKey() {
 }
 
 export async function verifyKeyTomato() {
     return verifyKey("_siyuanTomatoCode_")
 }
 
 export async function verifyKeyProgressive() {
-    return verifyKey("_siyuanProgressiveCode_")
+    _isValid.set(true);
+    return true;
 }
